tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/AuthorisedSteps.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using IdentityModel.Client;
+using TechTalk.SpecFlow;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
+{
+    [Binding]
+    public sealed class AuthorisedSteps
+    {
+        private readonly ScenarioContext _context;
+
+        public AuthorisedSteps(ScenarioContext context)
+        {
+            _context = context;
+        }
+
+        [Given(@"an authority user is logged in")]
+        public async Task GivenAnAuthorityUserIsLoggedIn()
+        {
+            var client = new HttpClient();
```

Comment:
see sam for different http client setup

Reply:
Spoke to Sam and we have decided not to implement this as it would be within the integration test steps, and to implment it would probably need a ticket